Sources/ConsoleKit/Output/ConsoleText.swift

```diff
@@ -23,6 +23,22 @@ extension String {
     }
 }
 
+extension Character {
+    /// Converts this `Character` to ``ConsoleText``.
+    ///
+    /// See ``ConsoleStyle`` for more information.
+    public func consoleText(_ style: ConsoleStyle = .plain) -> ConsoleText {
```

Comment:
If this just delegates to the `String` one I'd remove it to be honest, it's unnecessary overhead